telegram/ext/_application.py

```diff
@@ -959,8 +961,14 @@ def run_webhook(
                 header isn't set or it is set to a wrong token.
 
                 .. versionadded:: 20.0
-            unix (:class:`pathlib.Path` | :obj:`str`, optional): Path to the unix socket file. Path
-                does not need to exist, in which case the file will be created.
+            unix (:class:`pathlib.Path` | :obj:`str` | :class:`socket.socket`, optional): Either
```

Comment:
I find this a bit hard to read (very long sentences). Can we maybe make this a bullet point list? Something like

Either

* the path to a unix socket file as …. This will be passed to …. If the file does not exist …
* the socket object itself. This option allows …. Note that you will need …